evennia/contrib/game_systems/storage/tests.py

```diff
@@ -0,0 +1,122 @@
+from evennia.commands.default.tests import BaseEvenniaCommandTest
+from evennia.utils.create import create_object
+
+from . import storage
+
+
+class TestStorage(BaseEvenniaCommandTest):
+    def setUp(self):
+        super().setUp()
+        self.char1.location = self.room1
```

Comment:
Nitpick: I don't think this is needed; `char1` is already located in in `room1` from the onset afaik.